src/System.Management.Automation/SourceGenerators/PSVersionInfoGenerator/PSVersionInfoGenerator.cs

```diff
@@ -5,6 +5,8 @@
 using System.Globalization;
 using Microsoft.CodeAnalysis;
 
+#pragma warning disable RS1035
```

Comment:
A minor thing, how about add `<NoWarn>RS1035</NoWarn>` to the .csproj file?

Reply:
Fixed.. i like this better.